tests/unit/internal-session-test.js

```diff
@@ -385,7 +385,17 @@ describe('InternalSession', () => {
       return session.authenticate('authenticator');
     });
 
-    describe('when the authenticator resolves invaldiation', () => {
+    describe('when the authenticator resolves invalidation with params', () => {
+      it('when some data is sent with invalidate', () => {
```

Comment:
Test descriptions should always start with a verb actually, e.g. `it("passes the params on to the authenticator's invalidate method"`.